dashboard/config/scripts_json/allthethings.script_json

```diff
@@ -8325,6 +8383,18 @@
         "script.name": "allthethings",
         "activity_section.key": "f9c9c7a9-9f2c-48e1-867b-94d010f2795a"
       }
+    },
+    {
+      "seeding_key": {
+        "level.key": "AI Chat Test Level",
```

Comment:
are we missing this file from this PR? I tried to pull and seed locally and it can't find the level.

Reply:
Added `dashboard/config/scripts/levels/AI Chat Test Level.level`.